docs/reference/pages/model_reference.rst

```diff
@@ -780,3 +780,102 @@ Database fields
     .. attribute:: page
 
         (foreign key to ``Page``)
+
+``LogEntry``
+============
+
+Represents a record of an action performed on an object.
+
+Database fields
+~~~~~~~~~~~~~~~
+
+.. class:: LogEntry
+
+    .. attribute:: content_type
+
+        (foreign key to ``django.contrib.contenttypes.models.ContentType``)
+
+        A foreign key to the :class:`~django.contrib.contenttypes.models.ContentType` object that represents the specific model of this model.
+
+    .. attribute:: object_id
+
+        (text)
+
+        The object primary key.
+
+    .. attribute:: object_title
+
+        (text)
+
+        The object title at the time of the entry creation
+
+        Note: Wagtail will attempt to use ``get_admin_display_title`` or the string representation of the object passed to :meth:`~LogEntryManger.log_action`
+
+    .. attribute:: user
+
+        (foreign key to user model)
+
+        A foreign key to the user that triggered the action.
+
+    .. attribute:: data_json
+        (text)
+
+        The JSON representation of any additional details for each action.
+        e.g. source page id and title when copying from a page. Or workflow id/name and next step id/name on a workflow transition
+
+    .. attribute:: timestamp
+
+        (date/time)
+
+        The date/time when the entry was created.
+
+    .. attribute:: revision
+
+        (foreign key to :class:`PageRevision`)
+
+        Optional foreign key to the current page revision, when logging for :class:`Page` objects.
+
+    .. attribute:: created
```

Comment:
We probably don't need these anymore. Thanks to micrologging and inheritance (that we could implement later).

Just to get it in writing, the think the idea of inheritance we had was to allow actions to be given parents when they are registered. So for example:

 - `wagtail.copy` inherits `wagtail.create`
 - `wagtail_localize.copy_for_translation` inherits `wagtail.copy`
 - `wagtail.revert` inherits `wagtail.save_draft`

Then with some Python trickery, we turn this into a tree structure and make all queries looking for `wagtail.create` logs actually search for `wagtail.create`, `wagtail.copy` and `wagtail.copy_for_translation`.

Reply:
We do need `content_changed` and `deleted` for filtering purposes, and we discussed we'd leave all of them until we implement the "inheritance" idea.